Gemfile

```diff
@@ -26,6 +27,8 @@ gem 'paranoia'
 gem 'pg'
 gem 'puma'
 gem 'pundit'
+gem 'rails-i18n', '~> 5.0.0'
+gem 'ransack', github: 'activerecord-hackery/ransack'
```

Comment:
In your Gemfile, for the last officially released gem:
```ruby
gem 'ransack'
```

If you would like to use the latest updates (recommended), use the master branch:
```ruby
gem 'ransack', github: 'activerecord-hackery/ransack'
```